project/Publishing.scala

```diff
@@ -42,17 +42,16 @@ object Publishing {
           // Looks like `85-443a6fc-SNAP`
           Seq(version.value)
         case Release =>
-          // Looks like `85`, `85-443a6fc`
-          Seq(cromwellVersion, version.value)
+          // Looks like `85`, `85-443a6fc`, `latest`
+          Seq(cromwellVersion, version.value, "latest")
         case Debug =>
           // Ordinary local build with debug stuff
           // Looks like `85-443a6fc-DEBUG`
           Seq(version.value)
         case Standard =>
           // Merge to `develop`
-          // Looks like `85-443a6fc`, `latest`, `develop`
-          // TODO: once we automate releases, `latest` should move to `Release`
-          Seq(version.value, "latest", "develop")
+          // Looks like `85-443a6fc`, `develop`
+          Seq(version.value, "develop")
```

Comment:
The `dev` tag has active pulls on Docker Hub, but we inadvertently stopped updating it a while back. Seems like a good moment to address the issue.

```suggestion
          Seq(version.value, "dev", "develop")
```

![Screenshot 2025-05-07 at 12 09 45](https://github.com/user-attachments/assets/fe6839b7-73f9-4619-b2ad-fa1b34be1a23)